### ISSUE TITLE
REQUEST

### ISSUE THREAD

**Issue by WooAf** [AnthonyOSX/GO-Bot#216]
Would be nice if you store the settings in the folder directly so we can have more than one bot running with different settings without having to retype everything everytime we restart it. 

Thanks :)

:+1:

**Comment by MessiahMK** [comment]
Just hit Stop button, it will load for like 10 minutes, but eventually it will stop, then from that moment on your settings will be saved for every time you start the application. Peace be on you, young pokemon trainer.

**Comment by aeryl1999** [comment]
i think what we need is for the bot to stop discarding pokeballs

**Comment by rayn21** [comment]
I disagree.  You don't want to fill your inventory with regular poke balls when you are higher level.  It's better to set thresholds for each item that will keep, say, fifty of each ball on hand and discard potions.I have seen the bot throw very bot revealing errors when it initiates an encounter with no balls in inventory.  Keeping 50ish on hand is probably a happy medium.
